plugins/experimental/remap_stats/remap_stats.c

```diff
@@ -92,19 +92,20 @@ stat_add(char *name, TSMgmtInt amount, TSStatPersistence persist_type, TSMutex c
 static char *
 get_effective_host(TSHttpTxn txn)
 {
-  char *effective_url, *tmp;
   const char *host;
   int len;
   TSMBuffer buf;
   TSMLoc url_loc;
+  char *effective_url, *tmp;
```

Comment:
Why has this changed places? I don't think we should introduce changes that don't make things better. This is effectively a net zero change and I don't think we should make it. It just makes diff's harder to read.